Automattic/jetpack
# Publicize: Ensure newlines in custom messages are preserved

Fixes #23462

#### Changes proposed in this Pull Request:

* Replace `sanitize_text_field` with `sanitize_textarea_field` to preserve newlines.

#### Jetpack product discussion


N/A.

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:
##### Unit Tests
`jetpack docker phpunit -- --testsuite publicize `

##### Manual Testing
* Have a Publicize account connected
* Open the Classic editor
* Save a custom Publicize message with newlines
* Refresh the page and ensure the newlines are still present.
